### PR TITLE
[Stack a.1] Intended use OpenSaMD update

### PR DESCRIPTION
**Pull request by SimonBiggs**
After much discussion with potential customers I have found that there is significant market pull for a framework (monorepo?) that supports clinicians and researchers with taking their work (in-house software and/or research) and helping them turn it into a clinical medical device.

The possibility of using an open source business model for this could potentially help us change gear in the rate and quality of the software being deployed for clinical use in centres and hospitals around the globe.

Hence, the scope of this medical device is adjusting to instead be a platform that allows people to support clinicians and researchers to undergo the required software quality, review, and regulation in order to be able to supply their tools as software as a medical devices in hospitals and clinics around the world.

This is the beginnings of the documentation adjustments needed to support this approach.

### CODE REVIEW

File: docs/0-intended-use/autocontouring.md [hunk 1]
```diff
@@ -8,38 +8,13 @@ Original work by OpenRegulatory available at
...
-<!-- > Describe the core medical functionality of your device and how it treats,
-> diagnoses or alleviates a disease. Keep it high-level so that this
-> description is true for as long as possible even when the device is updated. -->
-
-Utilised as clinical decision support software within the radiotherapy
-treatment workflow. Provides anatomical contours utilised by health
-practitioners to create a radiotherapy treatment plan to assist in cancer
-treatment or other radiotherapy treatments.
+- Clinical decision support within radiotherapy via automated contouring of
+  both target and avoidance structures within the treatment planning workflow.
```
Comment (SimonBiggs) [inline docs/0-intended-use/autocontouring.md:17]: These removed sections are retained within the original document

File: docs/0-intended-use/index.md [hunk 2]
```diff
@@ -36,163 +36,39 @@ Original work by OpenRegulatory available at
...
+- Contraindications
+- Patient population
+- User profile
+- Use Environment including software/hardware
+- Operating principle
+
+The documents for these individual components are the following:
+
+- [](autocontouring.md)
+- [](sgrt.md)
```
Comment (SimonBiggs) [inline docs/0-intended-use/index.md:71]: The removed sections in this document have been moved into the new `autocontouring.md` document.

**Review by SimonBiggs — COMMENTED** [review]
I have added two comments below in an attempt to make the review easier.

File: docs/index.md [hunk 3]
```diff
@@ -39,32 +39,21 @@ to you with the software itself.
...
 - **Manufacturer Name**: {{device_manufacturer}}
 - **Manufacturer's Address**: 17 Grampian Place Tatton NSW 2650 Australia
-- **Software as a Medical Device Identification**: {{device_name}} {{device_version}}
+- **Software as a Medical Device Identification**: {{device_name}}
+  {{device_version}}
 
 ### Difference to the open source version
 
-This device corresponds to the software downloadable directly from
-<https://radiotherapy.ai/> (Regulated Medical Device). It does **not**
```
Comment (sjswerdloff) [inline docs/index.md:47]: The Web sites should be configuration items

File: docs/index.md [hunk 4]
```diff
@@ -39,32 +39,21 @@ to you with the software itself.
...
+clinical use, and is instead intended for research use only.
 
-The Regulated Medical Device described by this documentation corresponds only
-to the product that is able to be downloaded from <https://radiotherapy.ai/>.
-This is the product that is intended to be able to be used for clinical use
-according to the intended purpose described within this documentation.
-
-### Workflow overview
-
-Radiotherapy AI's {{device_name}} product (Regulated Medical Device) is
```
Comment (sjswerdloff) [inline docs/index.md:63]: The vendor should be a configuration item

**Review by sjswerdloff — CHANGES_REQUESTED** [review]
While it's not the end of the world for the items I mentioned to be hardcoded at the moment I think the purpose of this is to be a template.

File: docs/index.md (hunk 3 above)
Comment (SimonBiggs) [reply docs/index.md:47]: Yup, that was actually part of this change. This section (in red) is the text that has been removed. The updated line has the following:

https://github.com/OpenSaMD/OpenSaMD/blob/ccf95652a5a2334fee9f97558db77567efb57eaa/docs/index.md?plain=1#L48-L50

image

File: docs/index.md (hunk 4 above)
Comment (SimonBiggs) [reply docs/index.md:63]: Yup see above. This comment is on the removed text, not the updated text.

**Review by sjswerdloff — APPROVED** [review]

**Pull request merged**